src/Model/AuditReader.php

```diff
@@ -30,7 +30,11 @@ public function __construct(SimpleThingsAuditReader $auditReader)
 
     public function find(string $className, $id, $revisionId): ?object
     {
-        return $this->auditReader->find($className, $id, $revisionId);
+        try {
+            return $this->auditReader->find($className, $id, $revisionId);
+        } catch (\Exception $exception) {
```

Comment:
If we haven't specific domain exceptions for these situations, shouldn't we catch throwable?

Reply:
Done